HW2/P3/P3.txt

```diff
@@ -0,0 +1,22 @@
+Without AVX:
+
+1 thread:
+1074.652832 Million Complex FMAs in 5.11114192009 seconds, 210.256895387 million Complex FMAs / second
+
+2 threads:
+1074.652832 Million Complex FMAs in 2.72583699226 seconds, 394.246917571 million Complex FMAs / second
+
+4 threads:
+1074.652832 Million Complex FMAs in 1.60374307632 seconds, 670.090395316 million Complex FMAs / second
+
+
+With AVX:
+
+1 thread:
+1074.948291 Million Complex FMAs in 0.779345035553 seconds, 1379.29702758 million Complex FMAs / second
+
+2 threads:
+1074.948292 Million Complex FMAs in 0.668989896774 seconds, 1606.82290896 million Complex FMAs / second
+
+4 threads:
+1074.948294 Million Complex FMAs in 0.246887207031 seconds, 4354.00564868 million Complex FMAs / second
```

Comment:
Thats some amazing speedup your code shows since a jump from 2 to 4 threads is giving you a 3x performance increase.